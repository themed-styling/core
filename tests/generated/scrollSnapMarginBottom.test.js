import 'jest-styled-components'

import { scrollSnapMarginBottom } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(scrollSnapMarginBottom, 'scrollSnapMarginBottom')
