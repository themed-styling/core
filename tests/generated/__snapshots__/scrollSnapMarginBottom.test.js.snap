// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scrollSnapMarginBottom renders 0 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: 0;
  -moz-scroll-snap-margin-bottom: 0;
  -ms-scroll-snap-margin-bottom: 0;
  scroll-snap-margin-bottom: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 0;
  -moz-scroll-snap-margin-bottom: 0;
  -ms-scroll-snap-margin-bottom: 0;
  scroll-snap-margin-bottom: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 0;
  -moz-scroll-snap-margin-bottom: 0;
  -ms-scroll-snap-margin-bottom: 0;
  scroll-snap-margin-bottom: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 0;
  -moz-scroll-snap-margin-bottom: 0;
  -ms-scroll-snap-margin-bottom: 0;
  scroll-snap-margin-bottom: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders function 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: hasnot;
  -moz-scroll-snap-margin-bottom: hasnot;
  -ms-scroll-snap-margin-bottom: hasnot;
  scroll-snap-margin-bottom: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: has;
  -moz-scroll-snap-margin-bottom: has;
  -ms-scroll-snap-margin-bottom: has;
  scroll-snap-margin-bottom: has;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: hasnot;
  -moz-scroll-snap-margin-bottom: hasnot;
  -ms-scroll-snap-margin-bottom: hasnot;
  scroll-snap-margin-bottom: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: has;
  -moz-scroll-snap-margin-bottom: has;
  -ms-scroll-snap-margin-bottom: has;
  scroll-snap-margin-bottom: has;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders invalid theme path 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: invalid.path.1;
  -moz-scroll-snap-margin-bottom: invalid.path.1;
  -ms-scroll-snap-margin-bottom: invalid.path.1;
  scroll-snap-margin-bottom: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: invalid.path.1;
  -moz-scroll-snap-margin-bottom: invalid.path.1;
  -ms-scroll-snap-margin-bottom: invalid.path.1;
  scroll-snap-margin-bottom: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: invalid.path.1;
  -moz-scroll-snap-margin-bottom: invalid.path.1;
  -ms-scroll-snap-margin-bottom: invalid.path.1;
  scroll-snap-margin-bottom: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: invalid.path.1;
  -moz-scroll-snap-margin-bottom: invalid.path.1;
  -ms-scroll-snap-margin-bottom: invalid.path.1;
  scroll-snap-margin-bottom: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders negative number 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: -54881;
  -moz-scroll-snap-margin-bottom: -54881;
  -ms-scroll-snap-margin-bottom: -54881;
  scroll-snap-margin-bottom: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: -54881;
  -moz-scroll-snap-margin-bottom: -54881;
  -ms-scroll-snap-margin-bottom: -54881;
  scroll-snap-margin-bottom: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: -42365;
  -moz-scroll-snap-margin-bottom: -42365;
  -ms-scroll-snap-margin-bottom: -42365;
  scroll-snap-margin-bottom: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: -42365;
  -moz-scroll-snap-margin-bottom: -42365;
  -ms-scroll-snap-margin-bottom: -42365;
  scroll-snap-margin-bottom: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: July;
    -moz-scroll-snap-margin-bottom: July;
    -ms-scroll-snap-margin-bottom: July;
    scroll-snap-margin-bottom: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: September;
    -moz-scroll-snap-margin-bottom: September;
    -ms-scroll-snap-margin-bottom: September;
    scroll-snap-margin-bottom: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: July;
    -moz-scroll-snap-margin-bottom: July;
    -ms-scroll-snap-margin-bottom: July;
    scroll-snap-margin-bottom: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: September;
    -moz-scroll-snap-margin-bottom: September;
    -ms-scroll-snap-margin-bottom: September;
    scroll-snap-margin-bottom: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: May;
    -moz-scroll-snap-margin-bottom: May;
    -ms-scroll-snap-margin-bottom: May;
    scroll-snap-margin-bottom: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: June;
    -moz-scroll-snap-margin-bottom: June;
    -ms-scroll-snap-margin-bottom: June;
    scroll-snap-margin-bottom: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: April;
    -moz-scroll-snap-margin-bottom: April;
    -ms-scroll-snap-margin-bottom: April;
    scroll-snap-margin-bottom: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: May;
    -moz-scroll-snap-margin-bottom: May;
    -ms-scroll-snap-margin-bottom: May;
    scroll-snap-margin-bottom: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: June;
    -moz-scroll-snap-margin-bottom: June;
    -ms-scroll-snap-margin-bottom: June;
    scroll-snap-margin-bottom: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: April;
    -moz-scroll-snap-margin-bottom: April;
    -ms-scroll-snap-margin-bottom: April;
    scroll-snap-margin-bottom: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    -webkit-scroll-snap-margin-bottom: July;
    -moz-scroll-snap-margin-bottom: July;
    -ms-scroll-snap-margin-bottom: July;
    scroll-snap-margin-bottom: July;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-bottom: September;
    -moz-scroll-snap-margin-bottom: September;
    -ms-scroll-snap-margin-bottom: September;
    scroll-snap-margin-bottom: September;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    -webkit-scroll-snap-margin-bottom: July;
    -moz-scroll-snap-margin-bottom: July;
    -ms-scroll-snap-margin-bottom: July;
    scroll-snap-margin-bottom: July;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-bottom: September;
    -moz-scroll-snap-margin-bottom: September;
    -ms-scroll-snap-margin-bottom: September;
    scroll-snap-margin-bottom: September;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: November;
    -moz-scroll-snap-margin-bottom: November;
    -ms-scroll-snap-margin-bottom: November;
    scroll-snap-margin-bottom: November;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-bottom: August;
    -moz-scroll-snap-margin-bottom: August;
    -ms-scroll-snap-margin-bottom: August;
    scroll-snap-margin-bottom: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-bottom: January;
    -moz-scroll-snap-margin-bottom: January;
    -ms-scroll-snap-margin-bottom: January;
    scroll-snap-margin-bottom: January;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: December;
    -moz-scroll-snap-margin-bottom: December;
    -ms-scroll-snap-margin-bottom: December;
    scroll-snap-margin-bottom: December;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-bottom: April;
    -moz-scroll-snap-margin-bottom: April;
    -ms-scroll-snap-margin-bottom: April;
    scroll-snap-margin-bottom: April;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: May;
    -moz-scroll-snap-margin-bottom: May;
    -ms-scroll-snap-margin-bottom: May;
    scroll-snap-margin-bottom: May;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-bottom: June;
    -moz-scroll-snap-margin-bottom: June;
    -ms-scroll-snap-margin-bottom: June;
    scroll-snap-margin-bottom: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-scroll-snap-margin-bottom: January;
    -moz-scroll-snap-margin-bottom: January;
    -ms-scroll-snap-margin-bottom: January;
    scroll-snap-margin-bottom: January;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: December;
    -moz-scroll-snap-margin-bottom: December;
    -ms-scroll-snap-margin-bottom: December;
    scroll-snap-margin-bottom: December;
  }
}

@media January {
  .c0 {
    -webkit-scroll-snap-margin-bottom: April;
    -moz-scroll-snap-margin-bottom: April;
    -ms-scroll-snap-margin-bottom: April;
    scroll-snap-margin-bottom: April;
  }
}

@media April {
  .c0 {
    -webkit-scroll-snap-margin-bottom: May;
    -moz-scroll-snap-margin-bottom: May;
    -ms-scroll-snap-margin-bottom: May;
    scroll-snap-margin-bottom: May;
  }
}

@media June {
  .c0 {
    -webkit-scroll-snap-margin-bottom: June;
    -moz-scroll-snap-margin-bottom: June;
    -ms-scroll-snap-margin-bottom: June;
    scroll-snap-margin-bottom: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders postitive number 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: 54881;
  -moz-scroll-snap-margin-bottom: 54881;
  -ms-scroll-snap-margin-bottom: 54881;
  scroll-snap-margin-bottom: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 54881;
  -moz-scroll-snap-margin-bottom: 54881;
  -ms-scroll-snap-margin-bottom: 54881;
  scroll-snap-margin-bottom: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 42365;
  -moz-scroll-snap-margin-bottom: 42365;
  -ms-scroll-snap-margin-bottom: 42365;
  scroll-snap-margin-bottom: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: 42365;
  -moz-scroll-snap-margin-bottom: 42365;
  -ms-scroll-snap-margin-bottom: 42365;
  scroll-snap-margin-bottom: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders string 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: July;
  -moz-scroll-snap-margin-bottom: July;
  -ms-scroll-snap-margin-bottom: July;
  scroll-snap-margin-bottom: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: July;
  -moz-scroll-snap-margin-bottom: July;
  -ms-scroll-snap-margin-bottom: July;
  scroll-snap-margin-bottom: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: June;
  -moz-scroll-snap-margin-bottom: June;
  -ms-scroll-snap-margin-bottom: June;
  scroll-snap-margin-bottom: June;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: June;
  -moz-scroll-snap-margin-bottom: June;
  -ms-scroll-snap-margin-bottom: June;
  scroll-snap-margin-bottom: June;
}

<div
    className="c0"
  />,
]
`;

exports[`scrollSnapMarginBottom renders valid theme path 1`] = `
Array [
  .c0 {
  -webkit-scroll-snap-margin-bottom: DarkSlateBlue;
  -moz-scroll-snap-margin-bottom: DarkSlateBlue;
  -ms-scroll-snap-margin-bottom: DarkSlateBlue;
  scroll-snap-margin-bottom: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: DarkSlateBlue;
  -moz-scroll-snap-margin-bottom: DarkSlateBlue;
  -ms-scroll-snap-margin-bottom: DarkSlateBlue;
  scroll-snap-margin-bottom: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: DarkSlateBlue;
  -moz-scroll-snap-margin-bottom: DarkSlateBlue;
  -ms-scroll-snap-margin-bottom: DarkSlateBlue;
  scroll-snap-margin-bottom: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-scroll-snap-margin-bottom: DarkSlateBlue;
  -moz-scroll-snap-margin-bottom: DarkSlateBlue;
  -ms-scroll-snap-margin-bottom: DarkSlateBlue;
  scroll-snap-margin-bottom: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
